src/diffusers/loaders/lora_pipeline.py

```diff
@@ -1916,8 +2055,13 @@ def lora_state_dict(
                         f"The alpha key ({k}) seems to be incorrect. If you think this error is unexpected, please open as issue."
                     )
 
-        if return_alphas:
-            return state_dict, network_alphas
+        if return_alphas or return_lora_metadata:
+            outputs = [state_dict]
+            if return_alphas:
+                outputs.append(network_alphas)
+            if return_lora_metadata:
+                outputs.append(metadata)
+            return tuple(outputs)
```

Comment:
Just FYI, there is no need to convert to list and then back to tuple, you can "append" to tuples like so: `my_tuple += (item,)`. This creates a new tuple, as tuples are immutable. But it's fine like it is too.